architecture.md

```diff
@@ -125,21 +124,13 @@ Notes:
 * If it's via an API request, it involves an RPC request to the cluster main component.
 * `Cluster.PeerAdd()`
 * Figure out the real multiaddress for that peer (the one we see).
-* Let the `PeerManager` component know about the new peer. This adds it to the Libp2p host and notifies any component which needs to know
-about peers. This means also ability to perform RPC requests to that peer.
+* Broadcast the address to every cluster peer and add it to the host's libp2p peerstore. This means also ability to perform RPC requests to that peer.
```

Comment:
Typo "This means also ability to perform RPC requests to that peer" is grammatically off.  Perhaps "This gives each member of the cluster the ability to perform RPC  requests..."?